docs/read_geotif.md

```diff
@@ -0,0 +1,29 @@
+## Read Geo-tif Data
+
+Read in data (from tif format, most likely georeferenced image data). 
+
+**plantcv.geospatial.read_geotif**(*filename, mode="rgb"*)
+
+**returns** [PlantCV Spectral_data](https://plantcv.readthedocs.io/en/latest/Spectral_data/) object instance 
+
+- **Parameters:**
+    - filename - Filepath to .tif data 
+    - mode - Mode for geotif reading  
+
+- **Context:**
```

Comment:
Is 'context' important to include here?

Reply:
Totally not haha, probably kind of misleading and was leftover from copying another doc page. Removed!